Lib/multiprocessing/managers.py

```diff
@@ -188,10 +188,17 @@ def accepter(self):
                 c = self.listener.accept()
             except OSError:
                 continue
-            t = threading.Thread(target=self.handle_request, args=(c,))
+            t = threading.Thread(target=self._thread_handle_request, args=(c,))
             t.daemon = True
             t.start()
 
+    def _thread_handle_request(self, c):
```

Comment:
I don't know if Server.handle_request() is a public API, so I added a new private method to avoid any risk of regression. Tell me if handle_request() can be modified directly?

Reply:
I think it's code to modify `handle_request`.

Reply:
Ok, done.